scripts/sts-recreate.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+set -C -e -o pipefail
+##
+## Only run this script if you want to recreate statefulset: forbidden issue, revision problems, etc
```

Comment:
Copy editing suggestions. Does this still hold technically true?
## Only run this script if you want to recreate the statefulset: to deal with forbidden issues, 
revision problems, etc
## This script deletes the statefuset with cascading false (orphaning the pods) and forces 
the operator to recreate the statefulset once all the pods are recycled.

Reply:
happens sometime if we need to recycle statefulset

Reply:
fixed